### PR TITLE
CRM-16119: Fix Contact custom data multiple save/more buttons translation

### PR DESCRIPTION
**Pull request by mlutfy**

### CODE REVIEW

File: CRM/Contact/Form/CustomData.php [hunk 1]
```diff
@@ -185,11 +185,11 @@ public function buildQuickForm() {
         );
         if ($isMultiple) {
           $this->assign('multiRecordDisplay', $this->_multiRecordDisplay);
-          $saveButtonName = $this->_copyValueId ? 'Save a Copy' : 'Save';
+          $saveButtonName = $this->_copyValueId ? ts('Save a Copy') : ts('Save');
           $this->addButtons(array(
               array(
                 'type' => 'upload',
-                'name' => ts('%1', array(1 => $saveButtonName)),
```
Comment (colemanw) [inline CRM/Contact/Form/CustomData.php]: Wow, people sure do think of creative ways to misuse `ts`

Comment (colemanw) [inline CRM/Contact/Form/CustomData.php]: Not sure if you'll be surprised to learn grep turns up 8 other instances of `ts('%1'`

Comment (mlutfy) [inline CRM/Contact/Form/CustomData.php]: arg indeed, thanks for spotting those. I will try to fix a few more before the merge.

**Comment by mlutfy** [comment]
done

**Comment by mlutfy** [comment]
ok to test

**Comment by colemanw** [comment]
@mlutfy thanks for fixing those others!

**Comment by colemanw** [comment]
@mlutfy test failures are:

    PHP Parse error:  syntax error, unexpected '(', expecting ')' in CRM/Contact/Form/Edit/TagsAndGroups.php on line 73
    PHP Parse error:  syntax error, unexpected '[', expecting ']' in CRM/Event/Form/ParticipantFeeSelection.php on line 139

**Comment by mlutfy** [comment]
Oops.. thanks, I couldn't figure where the issue was. I reverted the fix in TagsAndGroups.
CRM/Event/Form/ParticipantFeeSelection.php seems to have had an unrelated issue, it seems to link correctly here?

**Comment by mlutfy** [comment]
And.. another oops.. I'm not used to rebasing, but visibly didn't rebase correctly. Closing this PR and will start again.

**Pull request closed without merging**